container-storage-setup.sh

```diff
@@ -23,7 +23,7 @@ set -e
 
 # This section reads the config file $INPUTFILE
 # Read man page for a description of currently supported options:
-# 'man docker-storage-setup'
+# 'man container-storage-setup'
 
 POOL_LV_NAME="docker-pool"
```

Comment:
This probably should be renamed to container-thin-pool? But this change will be tricky and we will have to make sure that it is not broken over upgrade. So this should kick in only for new thin pool set up.

Also for the case of default docker,  we should continue to use name docker-pool.